cmd/mesh/manifest-migrate.go

```diff
@@ -142,6 +212,6 @@ func migrateFromClusterConfig(rootArgs *rootArgs, mmArgs *manifestMigrateArgs, l
 	if err != nil {
 		return fmt.Errorf("error marshaling untyped map to YAML: %s", err)
 	}
-
-	return translateFunc(res, l)
+	// no concept of profile when translating in-cluster configs
```

Comment:
why not? in cluster CR still has a profile field.

Reply:
but I think the incluster values.yaml is a concatenation already? we can just translate from whatever it is

Reply:
Per our discussion, let's make everything relative to default profile / built-in values defaults in charts. This should be simpler and less complex for users to understand.